greenpithumb/moisture_sensor.py

```diff
@@ -1,5 +1,31 @@
+import adc
+
+#placeholder values
+_MIN_MOISTURE_LEVEL = 0.0
+_MAX_MOISTURE_LEVEL = 100.0
+
 class MoistureSensor(object):
     """Wrapper for DHT11 moisture sensor."""
 
+    def __init__(self, adc):
+    	"""Creates a new MoistureSensor wrapper.
+
+    	Args:
+    		adc: ADC(analog to digital) interface to receive 
+    		analog signals from moisture sensor.
+    	"""
+    	self._adc = adc
+
     def moisture(self):
-        raise NotImplementedError()
+        """Returns moisture level as percent."""
```

Comment:
Do you think it makes sense to convert it to a percentage? It doesn't look like it scales linearly, so it might be better just to leave it as a value of arbitrary scale.